settings.gradle

```
@@ -1,3 +1,19 @@
+plugins {
+  id 'com.gradle.develocity' version '3.17.5'
+}
+
+develocity {
+  buildScan {
+    termsOfUseUrl = 'https://gradle.com/terms-of-service'
```

```suggestion
    termsOfUseUrl = 'https://gradle.com/help/legal-terms-of-use'
```

URL recently changed:
https://docs.gradle.com/develocity/gradle-plugin/current/#connecting_to_scans_gradle_com